src/api-guides/request/deeplink-url-request.markdown

```diff
@@ -28,9 +27,9 @@ https://www.concursolutions.com/goto/travel-request/{requestUuid}?companyUUID={c
 
 |Name|Type|Format|Description|
 |---|---|---|---|
-`requestUuid`|`string`|`path`|**Required** The 20-character unique identifier of the request that is being read.|
-`companyUUID`|`string`|`query`|The unique identifier of the company, in UUID format, to which the request belongs to.|
-`context`|`string`|`query`|The access level of the user, which determines the form fields they can view/modify. Supported values: `TRAVELER`, `MANAGER`, or `PROCESSOR`|
+|`requestUuid`|`string`|`path`|**Required** The 32-character unique identifier of the request that is being read.|
+|`companyUUID`|`string`|`query`|The unique identifier of the company, in UUID format, to which the request belongs to.|
```

Comment:
it is possible to avoid mixing spelling with `Uuid` and `UUID`

Reply:
@laurentlhermine I can fix this for you.